test/jdk/java/awt/List/MouseDraggedOutCauseScrollingTest/MouseDraggedOutCauseScrollingTest.java

```diff
@@ -22,29 +22,29 @@
  */
 
 /*
-  test
+  @test
   @bug 6243382 8006070
   @summary Dragging of mouse outside of a List and Choice area don't work properly on XAWT
-  @author Dmitry.Cherepanov@SUN.COM area=awt.list
-  @run applet/manual=yesno MouseDraggedOutCauseScrollingTest.html
+  @requires (os.family == "linux")
+  @library /java/awt/regtesthelpers
+  @run main/manual MouseDraggedOutCauseScrollingTest
 */
```

Comment:
We usually move this and instructions block closer to a class declaration, for better readability.

Reply:
So people have been saying this, and I've even done it when asked, but I'm not sure I agree.
Although it has to go below the legal stuff, after that I like to see the test tag right at the top of the file,
not below a random number of imports.